ci/run_unit_tests.sh

```diff
@@ -3,12 +3,12 @@
 # On SQLite we run the tests in parallel.
 # This speeds things up, but is currently only safe for sqlite, since it uses
 # an in memory (and  therefore per-process) database.
+# submit coverage report only for postgres builds
 
 if [ $DB = sqlite ]; then
     extra_flags='-n auto'
```

Comment:
Noticed this because of it's proximity to the code you added -- the indentation here doesn't match the convention in the rest of the file, and other shell scripts.